src/pages/dataElements/form/useIsFieldValueUnique.ts

```diff
@@ -0,0 +1,55 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { useMemo } from 'react'
+import { Pager } from '../../../types/generated'
+
+const HAS_FIELD_VALUE_QUERY = {
+    dataElements: {
+        resource: 'dataElements',
+        params: (variables: Record<string, string>) => ({
+            pageSize: 1,
+            fields: 'id',
+            filter: [`${variables.field}:eq:${variables.value}`],
```

Comment:
Probably need the ID here as well (for edits), so if you're not getting an error when editing the "owner" of the unique value. 

Eg. ```ts
[`${variables.field}:eq:${variables.value}`, `id:ne:${variables.id}`, ],
```

Reply:
Fixed